spec/spec_helper.rb

```diff
@@ -17,6 +17,7 @@
     if Gem::Version.new(`puppet --version`) >= Gem::Version.new('3.5')
       Puppet.settings[:strict_variables]=true
     end
+    Puppet[:parser] = 'future' if ENV['PARSER'] == 'future'
```

Comment:
we don't need this if you specify FUTURE_PARSER="yes" as an environment variable in Travis.

Check out https://github.com/ghoneycutt/puppet-module-common/blob/master/.travis.yml#L20